### PR TITLE
Update OpenTelemetry parametric tests for .NET

### PR DESCRIPTION
**Pull request by bouwkast**
## Description

Unskip and update the parametric tests for OpenTelemetry for .NET.

## Motivation

Required for GA for OpenTelemetry

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com

### CODE REVIEW

**Comment by cbeauchesne** [comment]
Hi @bouwkast,

For linting, you can run the script `./format.sh` that do all the job for you.

**Comment by bouwkast** [comment]
Thanks @cbeauchesne I didn't know that (obviously 😄)
I think I ended up running `black .` which seemed to do the trick, but that'll help for the future

**Review by zacharycmontoya — COMMENTED** [review]
Whenever you're ready to open this PR up, I think it looks good

**Comment by bouwkast** [comment]
> Whenever you're ready to open this PR up, I think it looks good

Yep accidentally marked it ready before allowing the tests ran so redrafted it - now republished it

Failing tests look to be unrelated and all of the newly unskipped parametric ones for .NET are passing

**Review by zacharycmontoya — APPROVED** [review]

File: tests/parametric/test_otel_span_methods.py [hunk 1]
```diff
@@ -122,6 +110,13 @@ def test_otel_set_attributes_different_types(self, test_agent, test_library):
             assert root_span["meta"]["d_bool_val"] == "false"
             assert root_span["meta"]["array_val_int"] == "[10, 20]"
             assert root_span["meta"]["array_val_double"] == "[10.1, 20.2]"
+        elif root_span["meta"]["language"] == "dotnet":
+            assert root_span["meta"]["bool_val"] == "true"
+            assert root_span["meta"]["array_val_bool"] == "[true,false]"
+            assert root_span["meta"]["array_val_str"] == '["val1","val2"]'
```
Comment (marcotc) [inline tests/parametric/test_otel_span_methods.py:116]: Every otel sdk does string arrays differently I guess 😅

**Review by marcotc — APPROVED** [review]

**Review by pawelchcki — APPROVED** [review]

**Pull request merged**